privacyidea/static/components/user/controllers/userControllers.js

```diff
@@ -154,6 +154,60 @@ angular.module("privacyideaApp")
             });
         };
 
+        $scope.getEditableAttributes = function () {
```

Comment:
Maybe we should rename these functions to distinguish them from the factory functions?

Reply:
Will check and...

will do.

Reply:
We also have this name similarity e.g. for `getEditableResolvers`. So it is quite common in the UI to have a function name in the scope, that maps (or is the same name) in the factoriy.

I think I will not do this.